index.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  'all.schema-store': require('./schemas/all.schema-store.json'),
```

Comment:
is it necessary to expose it on API level?

Reply:
The reality is that it isn't. I will just need to modify the tests as they expect all schemas to be exposed.
Afaik nothing else will be needed.